### PR TITLE
Add route to activate actors

### PR DESCRIPTION
**Pull request by patriknw**
For a test scenario where a running test was hard killed, and tag_views are behind journal messages. The actors can be pinged to be loaded again, which should trigger continuation of writing to tag_views.

```
curl -X POST --data '{"name":"test-1620369501896","nrActors":1000 }' --header "content-type: application/json" http://127.0.0.1:8051/activate
```

Inspect journal with:

```
docker exec -it apt_cassandra cqlsh

select count(*) from akka_testing.messages;
select count(*) from akka_testing.tag_views;
```

It worked as expected when I tried. We can also try with akka-persistence-cassandra 1.0.3 because https://github.com/akka/akka-persistence-cassandra/pull/841 was fixed in 1.0.4 and that could be related.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,7 +3,7 @@ val AkkaPersistenceCassandraVersion = "1.0.4"
 val AkkaHttpVersion = "10.2.0"
 val AkkaProjectionVersion = "1.0.0"
 val AkkaManagementVersion = "1.0.9"
-val AkkaPersistenceJdbc = "4.0.0+57-17a7d41a" // will become 5.0
+val AkkaPersistenceJdbc = "5.0.0" // will become 5.0
```
Comment (octonato) [inline build.sbt]:
```suggestion
val AkkaPersistenceJdbc = "5.0.0"
```

File: src/main/scala/akka/projection/testing/ConfigurablePersistentActor.scala [hunk 2]
```diff
@@ -84,6 +86,9 @@ object ConfigurablePersistentActor {
                   ctx.self ! InternalPersist(totalEvents, testName, toPersist, bytesPerEvent, replyTo)
                 }
               }
+            case WakeUp(testName) =>
+              ctx.log.debug("WakeUp {}", ctx.self.path.name)
+              Effect.none
```
Comment (octonato) [inline src/main/scala/akka/projection/testing/ConfigurablePersistentActor.scala:91]: Not important, but we are not using `testName`. We can use either log it or make `WakeUp` an object.

**Review by octonato — APPROVED** [review]
LGTM

(left some nitpicking, nothing important)

**Comment by octonato** [comment]
This project is build the branch and the PR. We need to fix Travis. 

It is also good candidate to move to GH Actions because of containers.

**Comment by patriknw** [comment]
fails on jabba-install, we should migrate the CI

**Pull request merged**